packages/cds-annotation-parser/package.json

```diff
@@ -0,0 +1,63 @@
+{
+    "name": "@sap-ux/cds-annotation-parser",
+    "version": "0.1.0",
+    "description": "Annotation Parser for CDS",
+    "publisher": "SAPSE",
+    "author": "SAP SE",
+    "license": "Apache-2.0",
+    "main": "dist/index.js",
+    "scripts": {
+        "build": "npm-run-all clean compile",
+        "compile": "tsc --build ./tsconfig.build.json",
+        "clean:dist": "rimraf ./dist *.tsbuildinfo",
+        "clean": "rimraf ./reports",
+        "format:fix": "prettier --write --loglevel silent --ignore-path ../../../.prettierignore",
+        "format:fix:all": "prettier --write '**/*.{css,scss,html,js,json,ts,tsx,yaml,yml}' '!**/{out,dist,typings,node_modules}/**' '!**/*.{svg,png,xml}' --ignore-path ../../../.prettierignore",
+        "lint": "eslint . --ext .ts,.tsx",
+        "lint:fix": "eslint . --ext .ts --fix",
+        "lint:fix:all": "eslint . --ext .ts,.tsx --fix",
+        "lint:report": "eslint . --ext .ts,.tsx -f multiple ",
+        "lint:summary": "eslint . --ext .ts,.tsx -f summary",
+        "pre-commit": "lint-staged --quiet",
+        "test": "jest --maxWorkers=2",
+        "test:update": "node scripts/updateTestData.js",
+        "test:create": "node scripts/createTestData.js",
+        "watch": "tsc  --build ./ -watch"
+    },
+    "dependencies": {
+        "@sap-ux/odata-annotation-core": "workspace:*",
+        "@sap-ux/odata-vocabularies": "workspace:*",
+        "@sap-ux/odata-entity-model": "workspace:*",
+        "chevrotain": "7.1.1",
+        "vscode-languageserver-types": "3.17.2"
+    },
+    "devDependencies": {
+        "typescript": "4.9.5",
+        "npm-run-all": "4.1.5"
+    },
+    "files": [
+        "dist/"
+    ],
+    "jestSonar": {
+        "reportPath": "reports/test/unit",
+        "reportFile": "test-report.xml"
+    },
+    "eslint-formatter-multiple": {
+        "formatters": [
+            {
+                "name": "stylish",
+                "output": "console"
+            },
+            {
+                "name": "json",
+                "output": "file",
+                "path": "reports/lint/eslint.json"
+            },
+            {
+                "name": "checkstyle",
+                "output": "file",
+                "path": "reports/lint/eslint.checkstyle.xml"
+            }
+        ]
+    }
```

Comment:
```suggestion
```
Not needed here and can also be removed from the other annotation modules.

Reply:
Removed